### PR TITLE
chore: fix typo in WEATHER_LOCATION

### PR DESCRIPTION
**Pull request by lenntt**
breaking change
WEATHER_LOCAION -> WEATHER_LOCATION

Hope the trailing whitespace removal in the README isn't upsetting anyone.

### CODE REVIEW

File: Info-Orbs/include/widgets/weatherWidget.h [hunk 1]
```diff
@@ -44,7 +44,7 @@ class WeatherWidget : public Widget {
 
     WeatherDataModel model;
 
-    String weatherLocation = WEATHER_LOCAION;
+    String weatherLocation = WEATHER_LOCATION;
```
Comment (ericthelin) [inline Info-Orbs/include/widgets/weatherWidget.h:47]: There should be an ifdef in place to use the old string from someone's existing config.h

**Review by ericthelin — COMMENTED** [review]
Looks good with the exception of one small migration issue for existing config.h files.

**Comment by flattermann** [comment]
Please check the dev branch, I think most of the issues are already solved there.

**Comment by lenntt** [comment]
> Please check the dev branch, I think most of the issues are already solved there.

🤦 Theres a `dev` branch. Apologies.

**Pull request closed without merging**